hydra/functions/FlatteLineShape.h

```diff
@@ -121,12 +121,12 @@ namespace hydra {
         }
```

Comment:
L.110 : BaseFunctor<FlatteLineShape<CHANNEL, ArgIndex>, hydra::complex<double>, *7*>::operator=(other);**
This should be 1 -------------------------------------------------------------------------------------------^^^

This assignment operator can't compile either.